plugins/inputs/nfsclient/README.md

```diff
@@ -62,7 +62,8 @@ If `fullstat` is set, a great deal of additional metrics are collected, detailed
   - ops (integer, count) - The number of operations of this type executed.
   - retrans (integer, count) - The number of times an operation had to be retried (retrans = OP_trans - OP_ops.  See nfs_ops below)
   - exe (integer, miliseconds) - The number of miliseconds it took to process the operations.
-  - rtt (integer, miliseconds) - The round-trip time for operations.
+  - rtt (integer, miliseconds) - The total round-trip time for all operations.
+  - rtt_per_op (float, miliseconds) - The round-trip time per operation.
```

Comment:
When I first read this I thought it rtt_per_op would give an exact value for each operation, but after reading the code I understand this is an average rtt per op. It might be more clear for users to add in the docs that this is an average.

Reply:
agreed, done